ui/signed-out/src/commonMain/kotlin/social/androiddev/signedout/landing/LandingContent.kt

```diff
@@ -73,12 +71,12 @@ fun LandingContent(
     onGetStartedClicked: () -> Unit,
     appIcon: @Composable () -> Unit = {
         AsyncImage(
-            load = { loadImageIntoPainter(url = "https://via.placeholder.com/200x200/6FA4DE/010041?text=Dodo") },
-            painterFor = { remember { it } },
+            url = "https://via.placeholder.com/200x200/6FA4DE/010041?text=Dodo",
             contentDescription = "App Logo",
             modifier = Modifier
                 .padding(horizontal = 48.dp)
-                .size(240.dp)
```

Comment:
size already sets the width + height to the same value. Or is there a different reason for breaking this out?